Code/Framework/AzToolsFramework/AzToolsFramework/PaintBrushSettings/PaintBrushSettingsRequestBus.h

```diff
@@ -10,22 +10,25 @@
 
 #include <AzCore/Component/ComponentBus.h>
 #include <AzCore/EBus/EBus.h>
-#include <AzCore/std/containers/span.h>
-
-#include <AzToolsFramework/Viewport/ViewportTypes.h>
 
 namespace AzToolsFramework
 {
-    //! PaintBrushRequestBus is used to query the paintbrush for its settings and the current painted values while painting.
-    class PaintBrushRequests : public AZ::EBusTraits
+    class PaintBrushSettings;
+
+    //! PaintBrushSettingsRequestBus is used to get/set the global paintbrush settings
```

Comment:
nit: This should likely just be `PaintBrushSettingsRequests` (and also I'd suggest just removing the duplicated name - repeated elsewhere)